projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5008,6 +5009,25 @@ public static function authorize_starting() {
 		Jetpack_Options::update_option( 'unique_connection', $jetpack_unique_connection );
 	}
 
+	/**
+	 * This action fires when the site is registered (connected at a site level).
+	 */
+	public function handle_unique_registrations_stats() {
+		$jetpack_unique_registrations = Jetpack_Options::get_option( 'unique_registrations' );
```

Comment:
Have you considered utilizing the existing `unique_connections` option to save this as well?
Given it contains additional info, such as count of disconnects, it seems to me that this could fit there as well.

Reply:
Yes I did... but tbh I'm not sure that information saved on `unique_connections` is used for anything. Also, the option is just created upon user connection, so it would require a deeper change. I prefer not to touch that